MobileCenter/MobileCenterTests/MSMobileCenterTests.m

```diff
@@ -186,6 +186,23 @@ - (void)testStartServiceWithInvalidValues {
   XCTAssertEqual(servicesCount, [[MSMobileCenter sharedInstance] services].count);
 }
 
+- (void)testStartWithoutServices {
+  
+  // If
+  id logManager = OCMClassMock([MSLogManagerDefault class]);
+  OCMStub([logManager alloc]).andReturn(logManager);
+  OCMStub([logManager initWithAppSecret:[OCMArg any] installId:[OCMArg any] logUrl:[OCMArg any]]).andReturn(logManager);
+  
+  // Not allow processLog
+  OCMReject([logManager processLog:[OCMArg isKindOfClass:[MSStartServiceLog class]] forGroupId:[OCMArg any]]);
```

Comment:
I wonder if you forgot to add OCMVerify or something else to verify this OCMReject. I'm not familiar with this thing. 😄